addon/ng2/blueprints/route/index.js

```diff
@@ -1,16 +1,90 @@
-var stringUtils = require('ember-cli/lib/utilities/string');
+var fs = require('fs-extra');
+var path = require('path');
+var chalk = require('chalk');
 
+var imports, routeDefinitions;
+
+var simpleStringify = function (object) {
+  var simpleObject = {};
+  for (var prop in object) {
+    if (!object.hasOwnProperty(prop)) {
```

Comment:
Why not use `Object.keys(object)` (instead of `prop in object`) and avoid the `hasOwnProperty()` call ?